setup.py

```diff
@@ -1,13 +1,13 @@
 from setuptools import setup, find_packages
 
-version = '2.3.0'
+version = '2.5.0'
```

Comment:
Version number here should be changed to 2.4.0

Reply:
Right forgot to change that back